.gitmodules

```diff
@@ -1,3 +0,0 @@
-[submodule "ion-c"]
-	path = ion-c
-	url = https://github.com/amzn/ion-c.git
```

Comment:
🗺️ We no longer need to pull in `ion-c` as a submodule; we now provide a native implementation of `ion dump`.